GetVersionInfo() is not defined.
However, find_display_create_optional is always called with create=False.

--- libXgesture-0.1.1/src/gesture.c.orig	2013-05-07 03:10:03.000000000 +0200
+++ libXgesture-0.1.1/src/gesture.c	2020-08-28 17:50:35.806416204 +0200
@@ -106,11 +106,7 @@
     }
 
     if (!(dpyinfo = XextFindDisplay (gesture_info, dpy)) && create) {
-        dpyinfo = XextAddDisplay(gesture_info, dpy,
-                                 GESTURE_EXT_NAME,
-                                 &gesture_extension_hooks,
-                                 GestureNumberErrors,
-                                 (XPointer)GetVersionInfo(dpy));
+        return NULL; // never called
     }
 
     return dpyinfo;
